api-admin/src/main/resources/api/openapi.yml

```diff
@@ -156,7 +156,7 @@ paths:
         - { name: portalShortcode, in: path, required: true, schema: { type: string } }
         - { name: studyShortcode, in: path, required: true, schema: { type: string } }
         - { name: envName, in: path, required: true, schema: { type: string } }
-        - { name: id, in: path, required: true, schema: { type: string, format: uuid} }
+        - { name: id, in: path, required: true, schema: { type: string, format: uuid } }
```

Comment:
My IDE reformatted a bunch of stuff. They seem like good changes, but I'm happy to remove if we don't wanna include that.